### PR TITLE
feat(logs): added debug log messages

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- added debug log messages

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: pkg/kics/sink.go [hunk 1]
```diff
@@ -76,6 +77,7 @@ func (s *Service) sink(ctx context.Context, filename, scanID string, rc io.Reade
 		s.saveToFile(ctx, &file)
 	}
 	s.Tracker.TrackFileParse()
+	log.Debug().Msgf("Finishing to process file %s", filename)
```
Comment (nunoocx) [inline pkg/kics/sink.go:80]: Finished, instead of Finishing

File: pkg/engine/inspector.go [hunk 2]
```diff
@@ -227,6 +230,8 @@ func (c *Inspector) Inspect(
 			continue
 		}
 
+		log.Debug().Msgf("Finishing to load query %s after %v", queryMeta.Query, time.Since(queryStartTime))
```
Comment (nunoocx) [inline pkg/engine/inspector.go:233]: Finished, instead of Finishing

**Review by nunoocx — CHANGES_REQUESTED** [review]

**Review by nunoocx — DISMISSED** [review]
LGTM

**Review by nunoocx — APPROVED** [review]
LGTM

File: internal/console/pre_scan.go [hunk 3]
```diff
@@ -136,6 +139,18 @@ func (console *console) preScan() {
 	fmt.Println(versionMsg)
 	log.Info().Msgf(strings.ReplaceAll(versionMsg, "\n", ""))
 
+	log.Info().Msgf("Operating system: %s", runtime.GOOS)
+
+	mem, err := memory.Get()
+	if err != nil {
+		log.Info().Msgf("%s\n", err)
```
Comment (nunoocx) [inline internal/console/pre_scan.go:146]: Shouldn't be logged as error instead o f info, in this case?

**Review by nunoocx — CHANGES_REQUESTED** [review]

File: internal/console/pre_scan.go (hunk 3 above)
Comment (rafaela-soares) [reply internal/console/pre_scan.go:146]: Yes, my bad. On it

**Review by nunoocx — APPROVED** [review]
LGTM

**Pull request merged**